R/robustbayesianmetaanalysis.R

```diff
@@ -1288,10 +1292,10 @@ RobustBayesianMetaAnalysis <- function(jaspResults, dataset, options, state = NU
     type     = "individual",
     logBF    = options[["bayesFactorType"]] == "LogBF10",
     BF01     = options[["bayesFactorType"]] == "BF01",
-    include_theta = options[["results_theta"]]
+    include_theta = options[["resultsTheta"]]
```

Comment:
includeTheta?

Reply:
no, it's an argument to the `summary.RoBMA` function exported from the `RoBMA` package